src/components/sbb-datepicker/sbb-datepicker.e2e.ts

```diff
@@ -189,7 +189,7 @@ describe('sbb-datepicker', () => {
       await page.setContent(template);
```

Comment:
I'd appreciate if there is a test about aria-live messages (potentially you can copy something from my pull request)